### ISSUE TITLE
Accessibility: Missing labels when sending foto or video

### ISSUE THREAD

**Issue by heull001** [signalapp/Signal-Android#14034]
### Guidelines

- [x] I have searched searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] This isn't a feature request or a discussion topic

### Bug description

When sending a foto or video, many buttons are not labeled or the label is incomplete or misinterpretive.

- The button in chat to take a foto or video and send directly is labeled with "Direktaufnahme" in german. This could mean many different things, for example recording audio. "Foto oder Video aufnehmen" (like in the iOS-App) wold be a clear translation.
- On the screen to take a picture the first button is not labeled.
- The button to change between front and back camera is labeled, but the label should contain the selected camera, there is no way for visual impaired persons to know which is selected.
- On the review screen the only labeled button is "Add a message".

# Steps to reproduce

- enable TalkBack: https://support.google.com/accessibility/android/answer/6007100?hl=en
- open signal
- select an existing chat
- swipe right or left to "Toggle quick camara attachment drawer"
- doubletap to activate
- swipe right and left and doubletap to take a foto and send it, listen to the speech synthesizer

### Device

Google Pixel 8a

### Android version

15

### Signal version

7.35.0

**Comment by Sagar0-0** [comment]
Working on it
